tests/test_cli.py

```diff
@@ -293,6 +293,43 @@ def test_send_transaction_to_contract_from_compiled_contracts(
     assert is_0x_prefixed(transaction_hash)
 
 
+@pytest.mark.usefixtures("go_to_root_dir")
+def test_send_transaction_with_value_parameter(
+    runner, test_contract_address, test_contract_name
+):
+    """Change balance of a contract.
+
+    Test the transaction value parameter by a payable function on the
+    test contract. Because of the setup with an internal test RPC
+    endpoint, it is not possible to directly check the contracts
+    balance. Therefore an additional contract function is provided which
+    can be called to retrieve the contracts balance.
```

Comment:
How about we add another method to the deploy tools `balance` to get the balance of an account? This seems useful in conjunction with the ability to send tokens and would make the tests simpler.

Reply:
I feel unsure about that. It feels like replicating a basic functionality of web3 without additional value. And the only use case for our toolchain that comes into my mind is the funding of the bridge and then check if it has worked out. But therefore I rather would add another command for the `bridge-deploy` tool to fund the bridge, which then also checks the balance after a (configurable?) number of blocks.
How do you think about that?

Reply:
> It feels like replicating a basic functionality of web3 without additional value.

I feel like this about all of the deploy tools to be honest (or to be fair, at least if web3.py would be easier to use -- which might come with better ethpm integration).

But I see your point, I'm ok with not having this feature.